web/packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -361,6 +362,25 @@ export default class MainProcess {
       }
     );
 
+    ipcMain.handle(
+      'main-process-open-agent-logs-directory',
+      async (
+        _,
+        args: {
+          rootClusterUri: RootClusterUri;
+        }
+      ) => {
+        const { logsDirectory } = generateAgentConfigPaths(
+          this.settings,
+          args.rootClusterUri
+        );
+        const error = await shell.openPath(logsDirectory);
+        if (error) {
+          throw new Error(error);
+        }
```

Comment:
![go-in-js](https://github.com/gravitational/teleport/assets/27113/c7329787-f8f1-4fe9-a062-98d9a7a2bb17)

Reply:
😂